.github/workflows/build.yml

```
@@ -0,0 +1,248 @@
+          cd "${{ github.workspace }}\src"
+          Copy-Item mfaktc.ini ..
+          make SHELL="powershell.exe" -f Makefile.win
+
+      # MSVC 2019 on Windows 2019 has similar script, but it doesn't allows setting arch & host_arch and defaults to x86 (32 bit) env.
+      # So we have to run bat file for the env, but Make uses PowerShell afterward because it's much better at handling long paths &
+      # quotes when invoked from Make.
+      - name: Build from sources (cmd.exe with MSVC 2019)
+        if: ${{ matrix.sys.os == 'windows-2019' }}
+        shell: cmd
+        run: |
+          "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat" & cd src & copy mfaktc.ini .. & make SHELL="powershell.exe" -f Makefile.win
+
+      - name: Prepare build archive with description
+        shell: bash
+        run: |
+          choco install -y --no-progress zip
+          zip -9 -j "${{ env.base_name }}.zip" *
+          echo "[${{ env.base_name }}.zip](https://github.com/${{ github.repository }}/releases/download/${{ github.ref_name }}/${{ env.base_name }}.zip) | \
+          ${{ matrix.sys.cuda_version }} | ${{ steps.prepare.outputs.CC_MIN }}-${{ steps.prepare.outputs.CC_MAX }} | ${{ steps.prepare.outputs.OS_VER }} | \
+          ${{ steps.prepare.outputs.COMPILER_VER }} | ${{ steps.prepare.outputs.NVCC_VER }}" > ${{ env.base_name }}.txt
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ env.base_name }}
+          path: ${{ env.base_name }}.*
+# End job "build-win"
+
+# Begin job "upload-release"
+  upload-release:
+    if: startsWith(github.ref, 'refs/tags/')
+    needs: [ build-linux, build-win ]
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+
+      - name: Prepare list of release files & release notes
+        id: makeinfo
+        run: |
+          echo "Binary releases (automated builds) below." > RELEASE_NOTES.txt
+          echo "Compute Capability (CC) in the table means minimum and maximum versions supported." >> RELEASE_NOTES.txt
+          echo "CC versions are listed without the dot (i.e. 90 means 9.0 compute capability)." >> RELEASE_NOTES.txt
+          echo >> RELEASE_NOTES.txt
+          echo "File | CUDA version | Compute Capability | Build OS | Compiler version | NVCC version" >> RELEASE_NOTES.txt
+          echo "--- | --- | --- | --- | --- | ---" >> RELEASE_NOTES.txt
+          cat mfaktc-${{ github.ref_name }}-*-cuda*/mfaktc-${{ github.ref_name }}-*-cuda*.txt | sort -Vr >> RELEASE_NOTES.txt
+          echo 'RELEASE_FILES<<EOF' > $GITHUB_OUTPUT
+          ls -1 mfaktc-${{ github.ref_name }}-*-cuda*/mfaktc-${{ github.ref_name }}-*-cuda*.zip | sort -Vr >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
```

```suggestion
          {
            echo "Binary releases (automated builds) below."
            echo "Compute Capability (CC) in the table means minimum and maximum versions supported."
            echo "CC versions are listed without the dot (i.e. 90 means 9.0 compute capability)."
            echo
            echo "File | CUDA version | Compute Capability | Build OS | Compiler version | NVCC version"
            echo "--- | --- | --- | --- | --- | ---"
            sort -Vr mfaktc-${{ github.ref_name }}-*-cuda*/mfaktc-${{ github.ref_name }}-*-cuda*.txt
          } > RELEASE_NOTES.txt
          {
            echo 'RELEASE_FILES<<EOF'
            printf '%s\n' mfaktc-${{ github.ref_name }}-*-cuda*/mfaktc-${{ github.ref_name }}-*-cuda*.zip | sort -Vr
            echo 'EOF'
          } > $GITHUB_OUTPUT
```
Also, removed the useless `cat` and the useless `ls`.